__tests__/business-text/business-text.mutation.test.js

```diff
@@ -0,0 +1,292 @@
+              }
+          }
+      `,
+        variables: {
+          businessText: newBusinessText
+        },
+      })
+      .then(response => response)
+      .catch(e => e);
+
+    businessText = res.data.addBusinessText;
+    businessTextId = businessText._id;
+
+    expect(businessText)
+      .toHaveProperty('code', 'new-code');
+    expect(businessText)
+      .toHaveProperty('title', [
+        {
+          __typename: 'Language',
+          lang: 'uk',
+          value: 'НоваБТ',
+        },
+        {
+          __typename: 'Language',
+          lang: 'en',
+          value: 'NewBT',
+        },
+      ]);
+    expect(businessText.title)
+      .toBeInstanceOf(Array);
+    expect(businessText)
+      .toHaveProperty('text', [
+        {
+          __typename: 'Language',
+          lang: 'uk',
+          value: 'Тут бізнес текст',
+        },
+        {
+          __typename: 'Language',
+          lang: 'en',
+          value: 'Business text here',
+        },
+      ]);
+    expect(businessText.text)
+      .toBeInstanceOf(Array);
+    expect(businessText.code)
+      .isPrototypeOf(String);
+  });
+
+  test('#2 update business text', async () => {
+    const res = await client
+      .mutate({
+        mutation: gql`
+          mutation($id: ID!, $businessText: BusinessTextInput!) {
+              updateBusinessText(id: $id, businessText: $businessText) {
+                  ... on BusinessText {
+                      _id
+                      code
+                      title {
+                          value
+                          lang
+                      }
+                      text {
+                          value
+                          lang
+                      }
+                  }
+                  ... on Error {
+                      message
+                      statusCode
+                  }
+              }
+          }
+      `,
+        variables: {
+          id: businessTextId,
+          businessText: updatedBusinessText,
+        },
+      })
+      .catch(e => e);
+
+    businessText = res.data.updateBusinessText
+
+    expect(businessText)
+      .toHaveProperty(
+        'code',
+        'updated-code',
+      );
+    expect(businessText.title)
+      .toBeInstanceOf(Array);
+    expect(businessText)
+      .toHaveProperty('title', [
+        {
+          __typename: 'Language',
+          lang: 'uk',
+          value: 'ОновленаБТ',
+        },
+        {
+          __typename: 'Language',
+          lang: 'en',
+          value: 'UpdatedBT',
+        },
+      ]);
+    expect(businessText)
+      .toHaveProperty('text', [
+        {
+          __typename: 'Language',
+          lang: 'uk',
+          value: 'Оновлений бізнес текст',
+        },
+        {
+          __typename: 'Language',
+          lang: 'en',
+          value: 'Updated business text',
+        },
+      ]);
+  });
+
+  test('#3 update not existing businessText should return error', async () => {
```

Comment:
I think you also need the test where you try to update the business page with already existing code, but not its own code and that proper error message is returned